### PR TITLE
refactor: shared config to server side config

### PR DESCRIPTION
**Pull request by ArturMichalak**
## Description

The illegalActions configuration is only used on the server side.

## Checklist

- [X] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [X] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: fxmanifest.lua [hunk 1]
```diff
@@ -10,16 +10,21 @@ shared_scripts {
     '@qbx_core/modules/lib.lua',
 }
 
-client_script 'client/main.lua'
-server_script 'server/main.lua'
+client_scripts {
+    'config/client.lua',
```
[the remark this answers is not in the document]
Comment (mafewtm) [reply fxmanifest.lua:14]: It should remain in files. Files are not accessed by the server which is why we never include `'config/server.lua'` in files either.

https://docs.fivem.net/docs/scripting-reference/resource-manifest/resource-manifest/#file

[the remark this answers is not in the document]
Comment (ArturMichalak) [reply fxmanifest.lua:14]: So why can the server use shared.lua in this project?

[the remark this answers is not in the document]
Comment (Manason) [reply fxmanifest.lua:14]: It's a module, so it doesn't need to be explicitly loaded in the fxmanifest as the require keyword loads the file. The file itself needs to be sent to the client however, which is done in the fxmanifest using the files {} array

[the remark this answers is not in the document]
Comment (ArturMichalak) [reply fxmanifest.lua]: done

**Review by Manason — APPROVED** [review]

**Pull request merged**